webrtc/stun/stun_transport.nim

```diff
@@ -100,5 +93,5 @@ proc init*(
   var self = T(conn: conn, rng: rng)
   self.rng.generate(self.iceTieBreaker)
   self.readingLoop = stunReadLoop()
-  self.pendingConn = newAsyncQueue[StunConn](StunMaxPendingConnections)
+  self.pendingConn = newAsyncQueue[StunConn]()
```

Comment:
Why has it been removed? We can leave it configurable and check what other implementations use as the default value.

Reply:
I removed it because it'll only be used by nim-libp2p and nim-libp2p already offer this kind of limit. I'm not sure it was the good call, though.

Reply:
What limit in nim-libp2p are you referring to?

Reply:
Yeah, I'm not sure this project should make assumptions about libp2p.